MultimodalQnA/tests/test_compose_on_xeon.sh

```diff
@@ -20,8 +20,9 @@ export caption_fn="apple.txt"
 
 function build_docker_images() {
     cd $WORKPATH/docker_image_build
-    git clone https://github.com/opea-project/GenAIComps.git && cd GenAIComps && git checkout "${opea_branch:-"main"}" && cd ../
-
+    # TODO: Revert to the clone of opea-project "main" before merge
+    # git clone https://github.com/opea-project/GenAIComps.git && cd GenAIComps && git checkout "${opea_branch:-"main"}" && cd ../
+    git clone --single-branch --branch="mmqna-audio-query" https://github.com/mhbuehler/GenAIComps.git
```

Comment:
Please revert this back before PR merge.

Reply:
This is done